packages/features/webhooks/components/WebhookTestDisclosure.tsx

```diff
@@ -35,12 +31,24 @@ export default function WebhookTestDisclosure() {
           {t("ping_test")}
         </Button>
       </div>
-      <div className="p-4">
-        {!mutation.data && <em>{t("no_data_yet")}</em>}
-        {mutation.status === "success" && (
-          <div className="overflow-x-auto  text-gray-900">{JSON.stringify(mutation.data, null, 4)}</div>
-        )}
+      <div className="space-y-0 rounded-md border-0 border-neutral-200 sm:mx-0 md:border">
```

Comment:
border missing in mobile view 
![Screenshot 2023-01-31 at 10 38 46](https://user-images.githubusercontent.com/30310907/215805962-f3f7b70d-b0c1-4e54-9c0c-431d4201ed0f.png)

Reply:
done